<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigService</class>
 <widget class="QWidget" name="ConfigService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>230</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="enableService">
     <property name="toolTip">
      <string>If enabled the metadata extractor is called on newly indexed files.</string>
     </property>
     <property name="text">
      <string>Enable background Service</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Simultaneous calls:</string>
     </property>
     <property name="buddy">
      <cstring>kintspinbox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KIntSpinBox" name="kintspinbox">
     <property name="toolTip">
      <string>Number of simultaneous extractor calls. Default is 3.</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="documentServiceBox">
     <property name="toolTip">
      <string>Service needs to be restarted to take effect.</string>
     </property>
     <property name="text">
      <string>Fetch document information</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="musicServiceBox">
     <property name="toolTip">
      <string>Service needs to be restarted to take effect.</string>
     </property>
     <property name="text">
      <string>Fetch music information</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="videoServiceBox">
     <property name="toolTip">
      <string>Service needs to be restarted to take effect.</string>
     </property>
     <property name="text">
      <string>Fetch video information</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
